app/account/page.tsx

```
@@ -55,15 +56,19 @@ interface FormData {
- * Displays and manages the user's account page, allowing profile editing, avatar upload, password and email changes, two-factor authentication management, and account deletion.
- *
- * Renders profile information, account security settings, verification status, and provides interactive modals for updating sensitive information and enabling or disabling security features.
- */
-export default function AccountPage() {
-  const { user, profile, isLoading } = useAuth();
+function AccountDetails() {
+  const { user, profile } = useAuth();
+  const queryClient = useQueryClient();
+
+  const { data: accountData } = useSuspenseQuery({
+    queryKey: ["accountData", user!.id],
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)